agents/Taskfile.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+
+tasks:
+  build:
```

Comment:
maybe it's better to call it "use agent" which set corresponding folder as main workspace configs?

Reply:
`task build USE_AGENT=agents/example/minimax_v2v` something like this?

Reply:
no i mean `task use agents/example/minimax_v2v`